code/game/objects/structures/gym/weight_machine.dm

```diff
@@ -0,0 +1,131 @@
+/obj/structure/weightmachine
+	name = "chest press machine"
+	desc = "Just looking at this thing makes you feel tired."
+	icon = 'icons/obj/gym_equipment.dmi'
+	icon_state = "stacklifter"
+	base_icon_state = "stacklifter"
+	can_buckle = TRUE
+	density = TRUE
+	anchored = TRUE
+	blocks_emissive = EMISSIVE_BLOCK_UNIQUE
+
+	///How much we shift the user's pixel y when using the weight machine.
+	var/pixel_shift_y = -3
+
+	///The weight action we give to people that buckle themselves to us.
+	var/datum/action/push_weights/weight_action
+
+	///List of messages picked when using the machine.
+	var/static/list/more_weight = list(
+		"pushing it to the limit!",
+		"going into overdrive!",
+		"burning with determination!",
+		"rising up to the challenge!",
+		"getting strong now!",
+		"getting ripped!",
+	)
```

Comment:
ehhhh balloon alerts shouldn't really have exclamation points when they aren't failure states but i think these are cool enough to remain, just pointing it out since i saw it

Reply:
I think it's flavorful since it is kinda a big deal